lms/djangoapps/instructor/utils.py

```diff
@@ -164,20 +164,18 @@ def generate_course_forums_query(course_id, query_type, parent_id_check=None):
     return query
 
 
-def collect_ora2_data(course_id):
+def collect_ora2_data(course_id, include_email='False'):
     """
-    Query MySQL database for aggregated ora2 response data.
+    Query MySQL database for aggregated ora2 response data. If is_anonymous = false, display student username and emails.
     """
     cursor = get_read_replica_cursor_if_available(db)
-
     #Syntax unsupported by other vendors such as SQLite test db
     if db.connection.vendor != 'mysql':
         return '', ['']
 
-    raw_queries = ora2_data_queries().split(';')
-    
+    raw_queries = ora2_data_queries(include_email, course_id).split(';')
     cursor.execute(raw_queries[0])
-    cursor.execute(raw_queries[1], [course_id])
```

Comment:
I would actually prefer to keep this execute line as is.  It's always better to use SQL libraries directly for query parameterization rather than other string substitution methods (like .format below).  Otherwise, you are almost guaranteed to leave yourself open to some form of injection attack, because SQL escaping is really finicky and the generic Python string substitution methods won't handle all of the edge cases correctly (<a href="http://xkcd.com/327/">my favorite XKCD</a>).